### PR TITLE
take password interactively

### PR DESCRIPTION
**Pull request by vedhavyas**

### CODE REVIEW

**Comment by mikiquantum** [comment]
@vedhavyas what happened to codecov here?

**Comment by vedhavyas** [comment]
@mikiquantum previous develop build didn't run. Running it now. Lets wait and see

File: cmd/centrifuge/create_config.go [hunk 1]
```diff
@@ -30,11 +32,21 @@ func init() {
...
-			err := cmd.CreateConfig(targetDataDir, ethNodeURL, accountKeyPath, accountPassword, network, apiPort, p2pPort, bootstraps, txPoolAccess, false, "", nil)
+			_, err := fmt.Fprintln(os.Stderr, "Enter your Ethereum Account Password:")
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			pwd, err := terminal.ReadPassword(int(syscall.Stdin))
+			if err != nil {
+				// lets take empty password
+				log.Error(err)
```
Comment (vimukthi-git) [inline cmd/centrifuge/create_config.go]: what happens to the value of password here if there is a different error than empty password?

Comment (vedhavyas) [reply cmd/centrifuge/create_config.go]: its not empty password issue here. error happens when its not a terminal per se. like our scripts running the tests on travis etc...

Comment (vimukthi-git) [reply cmd/centrifuge/create_config.go]: but then `string(pwd)` will fail right? because `pwd` is `nil`

**Pull request merged**